src/ResourceManager/Compute/Commands.Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -275,8 +278,20 @@ public string Location
 
             var parameters = new Parameters(this, client);
 
+            // If the user did not specify a load balancer name, As in we are using a
+            // just
+            if (!loadBalancerNamePassedIn)
```

Comment:
@praries880 nit: we can consolidate all of the changes made in this file into one line:

```cs
LoadBalancerStrategy.IgnorePreExistingConfigCheck = String.IsNullOrWhitespace(LoadBalancerName);
```

Also, would you mind updating the comment above this line since it appears to be incomplete

Reply:
@sergey-shandar It behaves correctly, I have validated it multiple times on my desktop and while running the tests.